server/app.js

```diff
@@ -2,6 +2,7 @@ const express = require('express');
 const app = express();
 const bodyParser = require('body-parser');
 const sampledata = require('../sampledata.js');
+const db = require('../database/signup.js');
 
```

Comment:
@jGlass314 When you name this file, should we use something more specific like dbSignup? It seems like it will be an issue to name it just db since we're going to be requiring multiple database files for each query action.